src/tasks/publish/index.ts

```diff
@@ -73,6 +73,7 @@ export function setupPublishTask(): void {
       'Prevents contract compilation, deployment and artifact generation.'
     )
     .addFlag('verify', 'Automatically verify contract on Etherscan.')
+    .addFlag('force', 'Force publish without artifacts validation.')
```

Comment:
`force` still makes sense, but I wonder if we should've used something like `--skip-validation` to be clearer?

Reply:
I agree it makes more sense